bsdlib/doc/bsdlib_ug_adapting.rst

```diff
@@ -0,0 +1,360 @@
+.. _bsdlib_ug_adapting:
+
+User Guide: Adapting the BSD library to your OS
```

Comment:
I would prefer to say something like:
 * Creating an OS abstraction layer for your OS
 * Porting the BSD library to your OS

Reply:
Let's go for 'porting'.